pkg/middleware/util.go

```diff
@@ -27,6 +29,14 @@ func Gziper() macaron.Handler {
 			return
 		}
 
+		if strings.HasPrefix(requestPath, "/api/datasources/") && strings.Contains(requestPath, resourcesPath) {
+			return
+		}
+
+		if strings.HasPrefix(requestPath, "/api/plugins/") && strings.Contains(requestPath, resourcesPath) {
```

Comment:
Isn't it more concise/readable to combine this into the previous `if` statement with `(strings.HasPrefix(requestPath, "/api/datasources/") || strings.HasPrefix(requestPath, "/api/plugins/")`?

Reply:
Yeah good point :+1: